changes/get_pr_info.sh

```diff
@@ -120,7 +120,7 @@ $DEBUG && echoerr BASE_GITHASH="$BASE_GITHASH"
 #Bor's target branch -- use bors commit message as source of truth.
 if [[ "$BORS" == true ]] &&  [[ "$BRANCH" == "auto" || "$BRANCH" == "canary" ]] ; then
   commit_message=$(git log -1 --pretty=%B)
-  PR_NUMBER=$(echo "${commit_message}" | tail -1 | sed 's/Closes: #//')
+  PR_NUMBER=$(echo "${commit_message}" | grep 'Closes: #' | sed 's/Closes: #//')
```

Comment:
For cherry pick commits, the env var would eval to a list of PR numbers and would fail in subsequent logic below.  Here is a bad example https://github.com/diem/diem/commit/e8c8f206da0593edd519491b270d486e2108ed11

Reply:
Actually, only the last PR number matters.  Try append `tail -1` after the grep.

Reply:
thanks for the pointer, hope we've got all the edge cases.